Sources/Markdown/Markdown.docc/Markdown/VisitMarkup.md

```diff
@@ -0,0 +1,17 @@
+# Visiting Markup
+
+## Topics
+
+### Vistor
+
+- ``MarkupVisitor``
+
+### Walker
+
+``MarkupWalker`` is have a default implementation for ``MarkupVisitor``
```

Comment:
This is a little confusing to me. Did you mean

```suggestion
``MarkupWalker`` is a default implementation for ``MarkupVisitor``.
```

by chance?

Reply:
Yes, sorry for that. I'll self-review it again to make sure there is no such mistakes.